packages/next-swc/crates/next-build-test/benches/bench.rs

```diff
@@ -0,0 +1,161 @@
+use std::{
+    cmp::max,
+    time::{Duration, Instant},
+};
+
+use anyhow::Context;
+use criterion::{
+    black_box, criterion_group, criterion_main, measurement::ValueFormatter, Criterion,
+};
+use next_api::project::{ProjectContainer, ProjectOptions};
+use turbo_tasks::TurboTasks;
+use turbo_tasks_malloc::TurboMalloc;
+use turbopack_binding::turbo::tasks_memory::MemoryBackend;
+
+#[global_allocator]
+static ALLOC: turbo_tasks_malloc::TurboMalloc = turbo_tasks_malloc::TurboMalloc;
+
+/// Clones a repo down into a tmp folder and produces a ProjectOptions that can
+/// build it.
+fn init_bench(
+    src: &str,
+    install_command: &str,
+    package_path: Option<&str>,
+    next_config: String,
+    js_config: String,
+) -> (ProjectOptions, tempdir::TempDir) {
+    let tmp = tempdir::TempDir::new("turbo-bench").unwrap();
+
+    let clone = std::process::Command::new("git")
+        .arg("clone")
+        .arg("--depth")
+        .arg("1")
+        .arg(src)
+        .arg(tmp.path())
+        .status()
+        .unwrap();
+
+    std::process::Command::new("sh")
+        .arg("-c")
+        .current_dir(tmp.path())
+        .arg(install_command)
+        .status()
+        .unwrap();
+
+    let project_path = package_path
+        .map(|pp| tmp.path().join(pp))
+        .unwrap_or_else(|| tmp.path().to_owned());
+
+    (
+        ProjectOptions {
+            project_path: project_path.to_str().unwrap().to_owned(),
+            root_path: tmp.path().to_str().unwrap().to_owned(),
+            env: vec![],
+            define_env: next_api::project::DefineEnv {
+                client: vec![],
+                edge: vec![],
+                nodejs: vec![],
+            },
+            watch: false,
+            dev: false,
+            next_config,
+            js_config,
+        },
+        tmp,
+    )
+}
+
+fn do_in_runtime() -> impl FnOnce(
+    Pin<
+        Box<
+            dyn std::future::Future<Output = std::result::Result<(), anyhow::Error>>
+                + 'static
+                + Send
+                + Sync,
+        >,
+    >,
+) {
+    next_build_test::register();
+
+    let rt = tokio::runtime::Builder::new_multi_thread()
+        .enable_all()
+        .on_thread_stop(|| {
```

Comment:
Can't this just be

```
.on_thread_stop(TurboMalloc::thread_stop)
```

(and shouldn't clippy detect this? maybe I'm missing something?)